readme.md

```diff
@@ -12,8 +12,18 @@
 
 ```shell
 npm install -g fasp-server-cli
+
+then
```

Comment:
These steps are not required *in addition* to `npm install`. Why do you want to add them?